main.css

```diff
@@ -84,3 +84,14 @@ input[type="number"] {
 		transform:rotate(10deg)
 	}
 }
+
+@media only screen and (max-width: 768px) {
+	body {
+		max-width: 768px;
+	}
+	#wrapper {
+		margin-left: 5%;
+		margin-right: 5%;
+		width: 150%;
```

Comment:
Could you look into a more responsive way to do this instead of setting width to 150%?
You can restructure the original styling, if that helps